src/styles/_catalog.scss

```
@@ -1,9 +1,22 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
   max-width: 944px;
   margin: 0 auto;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, 200px);
+  gap: 48px;
+  justify-content: center;
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 1fr);
```

```suggestion
    grid-template-columns: repeat(2, 200px);
```
lets make gap not more than 48px in every case